src/main/java/com/hw/bookstore/exception/handler/ErrorBody.java

```diff
@@ -0,0 +1,18 @@
+package com.hw.bookstore.exception.handler;
+
+import java.util.ArrayList;
+import java.util.List;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@Getter
+@Setter
+@AllArgsConstructor
+@NoArgsConstructor
+public class ErrorBody {

```

Comment:
Why Error? Let's use Exception ;)

Reply:
Ohh, I use my exception response pattern, so I can manage my response format using ErrorBody. Let's keep this format